src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -252,9 +254,10 @@ function clearFocusTrap() {
 		<NcButton
 			:id="isNav ? triggerId : null"
 			ref="trigger-button-key"
-			class="header-menu__trigger"
 			:aria-controls="`header-menu-${id}`"
 			:aria-expanded="isOpened.toString()"
+			:aria-label
+			class="header-menu__trigger"
```

Comment:
Do you sort HTML attributes alphabetically? 🤨

Reply:
:abc: 

No but here I added the missing `aria-label`